cy-runner.yml

```diff
@@ -7,13 +6,8 @@ base:
     account: productusqa
     id: 2001459804
     domain: myvtex.com
+    urlExternalSeller: https://productusqaseller.myvtex.com
```

Comment:
does cy-runner throwing any error when this key is not present
@shashidhar85

Reply:
no